src/applications/auth/provider/PhabricatorGoogleIAPAuthProvider.php

```diff
@@ -95,7 +98,7 @@ protected function getLoginIcon() {
   }
 
   public function canAuthRequest(AphrontRequest $request) {
-    return $request->getHTTPHeader("x-goog-iap-jwt-assertion");
+    return $request->getHTTPHeader(PhabricatorGoogleIAPAuthProvider::IAP_HEADER);
```

Comment:
Okay, this looks good. I think Phabricator style is to explicitly cast this to a bool to make it clear you're not using the header value (see https://github.com/phacility/phabricator/blob/master/src/aphront/AphrontRequest.php#L792)

Reply:
Ah right - I read something on the internet suggesting that it's normally unnecessary in PHP, but agreed it's nice to be clear the function doesn't need to return an actual header.